eslint/babel-eslint-tests/test/integration/eslint/eslint.js

```diff
@@ -1879,3 +1881,23 @@ describe("verify", () => {
     );
   });
 });
+
+describe("ESLint config", () => {
```

Comment:
I slightly prefer to place them in a new test file `eslint/config.js` as the other tests in `eslint.js` are about verifying lint results.

Reply:
I don't have a strong opinion either way! Happy to move into a separate file.